alvr/server_core/src/connection.rs

```diff
@@ -312,8 +312,9 @@ pub fn handshake_loop(ctx: Arc<ConnectionContext>, lifecycle_state: Arc<RwLock<L
                 }
             };
 
-            if let WiredConnectionStatus::NotReady(m) = status {
-                dbg_connection!("handshake_loop: Wired connection not ready: {m}");
+            #[cfg_attr(not(debug_assertions), allow(unused_variables))]
+            if let WiredConnectionStatus::NotReady(s) = status {
```

Comment:
Could make this an expect ig

Reply:
Right. All allows should be turned into expect actually.